src/test/java/seedu/address/logic/commands/CommandTestUtil.java

```diff
@@ -65,6 +70,7 @@
 
     public static final String INVALID_IC_DESC = " " + PREFIX_IC + "A222223HH";
     public static final String INVALID_NAME_DESC = " " + PREFIX_NAME + "James&"; // '&' not allowed in names
+    public static final String INVALID_DOB_DESC = " " + PREFIX_DOB + "31/12/95"; // '-' should be used instead of '/'
```

Comment:
Maybe it is better to use the full class name for clarity?
A good case of invalid check since '/' is quite a common way of inputting dates.